v2/support/demo_get_set_app_metadata/assets/iframe.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <script src="https://cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js"></script>
+</head>
+<body>
+  <h3>Apps framework metadata test:</h3>
+  <label for="formName">Enter form name to save:</label>
+  <input style="width:250px" name="formName" id="formName" value="">
+  </br>
+  <input type="checkbox" id="myCheckbox">My checkbox</br>
+  <hr>
+  <button id=getMetaData>Get metadata</button>
+  </br>
+  <button id="setMetaData">Set metadata</button>
+  <p>NOTE: Remember to select 'Reload all apps' to retrieve newly saved values.</p>
+  <script src="https://static.zdassets.com/zendesk_app_framework_sdk/2.0/zaf_sdk.min.js"></script>
```

Comment:
You can move this script up to line 6 above.

Reply:
Thanks for doing this Hoang. I feel embarrassed for not getting around to this until now.

For this particular comment -- did you mean to move 18 up to the header block? The only reason I put it in Body is that's where `zat new` puts it.